tools/gfx/transient-resource-heap-base.h

```diff
@@ -45,6 +47,26 @@ class TransientResourceHeapBaseImpl : public TransientResourceHeapBase
         return (value + alignment - 1) / alignment * alignment;
     }
 
+    Result allocateStagingBuffer(size_t size, IBufferResource*& outBufferWeakPtr, ResourceState state)
```

Comment:
Not in this PR, but alignUp might be better off being `(value + alignment - 1) & ~(alignment - 1)`. Assuming alignment is power of 2.